Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Possible_External_RDP_Brute-Force_-_Set_Verdict.yml

```diff
@@ -0,0 +1,557 @@
+id: Cortex XDR - Possible External RDP Brute-Force - Set Verdict
```

Comment:
Source Geolocation - Connection from an unusual country. - need to rephrase it to be more clear, something like:
Source Geolocation - Checks if the source geo-location has been seen in the past communicating with the brute-forced service.
* For all of them.

Reply:
to be accurate, it should detail the condition that the element will suspicious. I will rephrase it in this direction